pkg/odo/cli/component/common_link.go

```diff
@@ -142,7 +134,12 @@ func (o *commonLinkOptions) complete(name string, cmd *cobra.Command, args []str
 				Group:    deploymentSelfLinkSplit[2], // "apps" in above example output
 				Version:  deploymentSelfLinkSplit[3], // "v1" in above example output
 				Resource: deploymentSelfLinkSplit[6], // "deployments" in above example output
-			},
+			}
+		}
+
+		// Populate the application selector field in service binding request
+		o.serviceBinding.Spec.Application = &servicebinding.Application{
+			Ref: ref,
```

Comment:
If I add `,` as [pointed here](https://github.com/openshift/odo/pull/4554/files#r602136902) I get below build error:
```
pkg/odo/cli/component/common_link.go:142:4: unknown field 'Ref' in struct literal of type "github.com/redhat-developer/service-binding-operator/pkg/apis/operators/v1alpha1".Application
```

Reply:
If I replace `Ref` with `LocalObjectReference` (I used `Ctrl+Space` to see what the IDE suggests), it complains that `ref` is undefined:
```sh
pkg/odo/cli/component/common_link.go:142:26: undefined: ref
```